src/api/app/components/workflow_run_filter_component.html.haml

```diff
@@ -8,40 +8,52 @@
                             <p>or a git commit hash like
                             <em>97561db8664eaf86a1e4c7b77d5fb5d5bff6681e</em></p>'
 
-.list-group.list-group-flush.my-2
-  = render WorkflowRunFilterLinkComponent.new(token: @token, text: 'All', amount: @count['all'],
-                                              filter_item: {}, selected_filter: @selected_filter)
-.list-group.list-group-flush.mt-5.mb-2
-  %h5.mx-3
-    Status
-    = render partial: 'webui/shared/info_popover', locals: { position: 'bottom', text: status_info_text }
-  = render WorkflowRunFilterLinkComponent.new(token: @token, text: 'Succeeded', amount: @count['success'], icon: 'fas fa-check text-primary',
-                                              filter_item: { status: 'success' }, selected_filter: @selected_filter)
-  = render WorkflowRunFilterLinkComponent.new(token: @token, text: 'Running', amount: @count['running'], icon: 'fas fa-running',
-                                              filter_item: { status: 'running' }, selected_filter: @selected_filter)
-  = render WorkflowRunFilterLinkComponent.new(token: @token, text: 'Failed', amount: @count['fail'], icon: 'fas fa-exclamation-triangle text-danger',
-                                              filter_item: { status: 'fail' }, selected_filter: @selected_filter)
-.list-group.list-group-flush.mt-5.mb-2
-  %h5.mx-3
-    Event Type
-    = render partial: 'webui/shared/info_popover', locals: { position: 'bottom', text: event_type_info_text }
-  = render WorkflowRunFilterLinkComponent.new(token: @token, text: 'Pull/Merge Request', amount: @count['pull_request'],
-                                              filter_item: { generic_event_type: 'pull_request' }, selected_filter: @selected_filter)
-  = render WorkflowRunFilterLinkComponent.new(token: @token, text: 'Push', amount: @count['push'],
-                                              filter_item: { generic_event_type: 'push' }, selected_filter: @selected_filter)
-  = render WorkflowRunFilterLinkComponent.new(token: @token, text: 'Tag Push', amount: @count['tag_push'],
-                                              filter_item: { generic_event_type: 'tag_push' }, selected_filter: @selected_filter)
-.list-group.list-group-flush.mt-5.mb-2
-  %h5.mx-3
-    Event Source
-    = render partial: 'webui/shared/info_popover', locals: { position: 'bottom', text: event_source_info_text }
-  = render WorkflowRunFilterInputComponent.new(token_id: @token.id, text: 'PR/MR', selected_input_filter: @selected_filter[:pr_number],
-                                               placeholder: '12345')
-  = render WorkflowRunFilterInputComponent.new(token_id: @token.id, text: 'Commit', selected_input_filter: @selected_filter[:commit_sha],
-                                               placeholder: '97561db8664eaf86a1e4c7b77d5fb5d5bff6681e')
+-# TODO: fix the form parameters
+= form_for(@token, url: token_workflow_runs_path(@token, @selected_filter), method: :get) do |form|
```

Comment:
In my tests, the `@selected_filter` was not needed because the inputs were sent via query strings, and those selected filters were just adding noise

Reply:
True. It's no longer needed now. Dropped
(It will make much more sense for the whole picture when frontend and backend changes will be put together)